std/uni.d

```diff
@@ -2713,8 +2713,11 @@ private:
             return CodepointInterval(a, b);
         }
 
+        enum modifiable = !is(ElementType!Range == const(T), T);
```

Comment:
```d
    auto arr = [immutable int(1), 2, 3];
    alias Range = typeof(arr);
    pragma(msg, is(ElementType!Range == const(T), T)); // false
```

I thinks [`hasAssignableElements!Range`](https://dlang.org/library/std/range/primitives/has_assignable_elements.html) would be a better fit ;-)